usaspending_api/recipient/migrations/0017_recipientprofile_parent_uei.py

```diff
@@ -0,0 +1,18 @@
+# Generated by Django 2.2.17 on 2022-03-10 18:42
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('recipient', '0016_auto_20220307_1946'),
+    ]
+
+    operations = [
+        migrations.AddField(
+            model_name='recipientprofile',
+            name='parent_uei',
+            field=models.TextField(db_index=True, null=True),
```

Comment:
So why add this? It doesn't appear to be used at all in this PR ..., since the `temporary_recipients_from_transactions_view.parent_uei` field seems to be the source for `parent_uei` in this PR and that comes from ...

```
COALESCE(fpds.ultimate_parent_uei, fabs.ultimate_parent_uei) AS parent_uei,
```

If it is needed, see comment/question about indexing below.

Reply:
We are adding the `parent_uei` field to the recipient_profile table itself, so I added an index to it,which to be fair, might not be required, so I don't mind removing it if it seems unnecessary. As to why we are adding `parent_uei` to the table - the ticket specifically said to do it.
```
With either of the above options, it would be a good idea to include parent_uei in the recipient_profile table. This should be added to the temporary_recipients_from_transactions_view temporary_restock_recipient_profile and recipient_profile
```